generators/client/templates/vue/src/main/webapp/app/shared/alert/alert.service.ts.ejs

```diff
@@ -0,0 +1,61 @@
+import Vue from 'vue';
+
+export default class AlertService {
```

Comment:
is it possible to have alert.service.spec.ts too plz ?

Reply:
added for translation/entity-header scenarios in 02c6214  :)